docs/docs/steps/params/README.md

```diff
@@ -0,0 +1,110 @@
+# Params
```

Comment:
This looks like an old version of this file that references the v1 `params` object. Was this included intentionally?